spec/services/by_team_project_fetcher_service_spec.rb

```diff
@@ -49,6 +49,62 @@
     end
   end
 
+  describe "#new" do
+    before do
+      Timecop.freeze(Time.local(2023, 1, 1, 0, 0))
```

Comment:
We don't need to use Timecop as Rspec has built-in time helpers already. Have a look at other tests which use `travel_to`, e.g. /spec/requests/all/by_month/projects_controller_spec.rb#L20